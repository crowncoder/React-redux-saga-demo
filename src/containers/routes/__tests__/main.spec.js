import React from 'react';
import { render } from 'enzyme';
import Main from '../main';
it('Render Main', () => {
	const wrapper = render(<Main />);
	expect(wrapper).toMatchSnapshot();
});
