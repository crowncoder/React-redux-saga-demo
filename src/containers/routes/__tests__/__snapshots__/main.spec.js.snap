// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Main 1`] = `
<div>
  <div
    class="userImage"
  />
  <div
    class="settingImage"
  />
  <article>
    <header>
      <h1
        class="welcome"
      >
        Welcome to GTC's Demo
      </h1>
    </header>
  </article>
</div>
`;
